package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "dependencies": {
     "@edx/edx-bootstrap": "0.4.3",
-    "@edx/paragon": "1.7.1",
+    "@edx/paragon": "2.5.6",
```

Comment:
This is a major version upgrade, does this break anything else?

Reply:
Good question. Paragon does "semantic versioning" which for them means that major version bumps imply a breaking change. The changelog for v2.0.0 is: https://github.com/edx/paragon/releases/tag/v2.0.0 -- the only change is dropping defaultChecked for CheckBox. I verified with Jae that that was the only thing to look out for, and we don't use defaultChecked in edx-platform.